packages/api/app/Controllers/Http/UploadController.js

```diff
@@ -58,15 +58,13 @@ class UploadController {
 
 			uploads = await Promise.all(
 				files.movedList().map((file) =>
-					user.uploads().create(
-						{
-							filename: file.fileName,
-							object: objectInfo.object,
-							object_id: objectInfo.object_id,
-						},
-						trx,
-					),
+					user.uploads().create({
+						filename: file.fileName,
+						object: objectInfo.object,
+						object_id: objectInfo.object_id,
+					}),
 				),
+				trx,
```

Comment:
Mudei o trx para o Promise.all() porque o catch vai apagar todos os arquivos enviados caso algum for invalido.

Reply:
Não entendi muito bem aqui. O promise.all não recebe uma transaction. Além disso o catch vai apagar os registros físicos e não os registros no db.